app/controllers/candidate_interface/degrees/destroy_controller.rb

```diff
@@ -0,0 +1,23 @@
+module CandidateInterface
+  class Degrees::DestroyController < CandidateInterfaceController
+    def confirm_destroy
+      application_from = current_candidate.current_application
+      @degree = DegreesForm.find_using_application(application_from, degrees_params[:id])
```

Comment:
`find_by_application` sounds better than `find_using_application`, doesn't it?
actually `find_using_application_id`
... actually this is not just a find 😱

Reply:
Updated it to `find_by_application`.